### PR TITLE
First batch around spanish cities

### PR DESCRIPTION
**Pull request by inouire**
This batch is small, testing that everything works fine

### CODE REVIEW

File: stations.csv [hunk 1]
```diff
@@ -4891,7 +4891,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;is_city;country;
 6489;Londoño;londono;7131406;;;;;f;ES;f;Europe/Madrid;f;ESADJ;;f;;f;;f;;f;;f;;f;;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
 6490;Adanero (Muñoz);adanero-munoz;7110407;;;;;f;ES;f;Europe/Madrid;f;ESADK;;f;;f;;f;;f;;f;;f;;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
 6491;Medellín;medellin;7137408;;;;;f;ES;f;Europe/Madrid;f;ESADL;;f;;f;;f;;f;;f;;f;;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
-6492;Aranda de Duero – Montecillo;aranda-de-duero-montecillo;7117200;;41.667778;-3.695836;;f;ES;f;Europe/Madrid;t;ESADM;;f;;f;;f;;f;;f;;f;;;f;;f;17200;t;;f;f;;;;;;;;;;;;;;;;;;
```
Comment (mckbrmn) [inline stations.csv]: What's the logic for using an en-dash here vs a regular dash for other Spanish stations?

File: stations.csv [hunk 2]
```diff
@@ -4998,7 +4998,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;is_city;country;
 6596;Guadalajara;guadalajara;7109301;;40.632399;-3.160243;;f;ES;f;Europe/Madrid;t;;;f;;f;;f;;f;;f;;f;;;f;;f;GUADA;t;;f;f;;;;;;;;;;;;;;;;;;
 6597;Arenys de Mar;arenys-de-mar;7109383;;;;;f;ES;f;Europe/Madrid;f;ESAHS;;f;;f;;f;;f;;f;;f;;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
 6598;El Ferrol;el-ferrol;7121010;;43.488266;-8.230855;;f;ES;f;Europe/Madrid;t;ESAHT;;f;;f;7100162;f;;f;;f;;f;;;f;;f;21010;t;;f;f;;;;;;;;;;;;;;;;;;
-6599;Betanzos – Cidade;betanzos-cidade;7121001;;43.284264;-8.215673;;f;ES;f;Europe/Madrid;t;ESAHU;;f;;f;7100237;f;;f;;f;;f;;;f;;f;21001;t;;f;f;;;;Ciudad;;;;;;;;;;;;;;
+6599;Betanzos – Cidade;betanzos-cidade;7121001;;43.284264;-8.215673;27017;f;ES;f;Europe/Madrid;t;ESAHU;;f;;f;7100237;f;;f;;f;;f;;;f;;f;21001;t;;f;f;;;;Ciudad;;;;;;;;;;;;;;
```
Comment (mckbrmn) [inline stations.csv]: Same here

File: stations.csv (hunk 1 above)
Comment (inouire) [inline stations.csv]: Nothing special, I just kept the existing name. Should I use regular dash everywhere ?

File: stations.csv (hunk 2 above)
Comment (inouire) [inline stations.csv]: Same as above

File: stations.csv (hunk 1 above)
Comment (mckbrmn) [inline stations.csv]: I say yes, use regular dash. I cannot find any instances of the name written in that manner.

File: stations.csv (hunk 2 above)
Comment (mckbrmn) [inline stations.csv]: Same as previously answered.

**Comment by inouire** [comment]
Fixed the dashes :dash:

**Comment by mckbrmn** [comment]
I meant for you to also remove the extraneous spaces

**Comment by inouire** [comment]
Fixed the spaces :space_invader:

**Pull request merged**